examples/mongocxx/mongodb.com/documentation_examples.cpp

```diff
@@ -1180,13 +1182,62 @@ void delete_examples(mongocxx::database db) {
     }
 }
 
+void start_versioned_API_examples() {
+
+    // To start a versioned API, you can set specific options in the mongocxx::client's
+    // server flags:
+    auto server_api = mongocxx::options::server_api(mongocxx::options::server_api::version::k_version_1);
+
+    // There are several helper member functions for converting between strings and API versions:
+    auto version_string = mongocxx::options::server_api::version_to_string(mongocxx::options::server_api::version::k_version_1);
+
+    // Convert a string to an API version:
+    
+    // Using an invalid version string should throw a mongocxx::logic_error:
+    try
+     {
+         auto version = mongocxx::options::server_api::version_from_string(R"({"version" : "1"}")");
+     }
+    catch(const mongocxx::logic_error&)
+     {
+	 std::cout << "calling version_from_string() with an invalid version string threw an error, as expected: good\n";
+     }
+
+    // Using a valid version string should be ok:
+    {
+         auto version = mongocxx::options::server_api::version_from_string(R"({"version" : "1"}")");
```

Comment:
Ooops, looks like if I want to show how to handle an invalid version, I should also show what happens with a valid one! Will correct.